### PR TITLE
EN6-127 Fixed response body for DELETE ContentType method

### PR DESCRIPTION
**Pull request by ffleandro**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/web/content/ContentTypeResourceController.java [hunk 1]
```diff
@@ -76,10 +92,10 @@ private void validateCreate(ContentTypeDtoRequest contentType, BindingResult bin
 
     @Override
     @RestAccessControl(permission = Permission.SUPERUSER)
-    public ResponseEntity<Void> delete(@PathVariable("code") String code) {
+    public ResponseEntity<SimpleRestResponse<Map>> delete(@PathVariable("code") String code) {
         logger.debug("REST request - delete content type {}", code);
         service.delete(code);
-        return ResponseEntity.ok().build();
+        return ResponseEntity.ok(new SimpleRestResponse<>(new HashMap<>()));
```
Comment (eugeniosant) [inline plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/web/content/ContentTypeResourceController.java]: This fix is incomplete. The api must return the code of the deleted content type. In particular the response body have to be something like this:
`
{
    "payload": {
        "code": "<contentTypeCode>"
    },
    "errors": [],
    "metaData": {}
 }
`

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by joewhite101** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by joewhite101** [comment]
Retest this please

**Comment by entando-service** [comment]
All is Well

**Comment by ffleandro** [comment]
@joewhite101 Eugenio had left a comment on this PR. Should I reopen the PR?

**Comment by joewhite101** [comment]
@ffleandro yeah or create a new one to resolve. You can re-open the branch and create a new PR if you want.

**Pull request merged**